keras/layers/wrappers.py

```diff
@@ -208,21 +208,30 @@ def compute_output_shape(self, input_shape):
         elif self.merge_mode is None:
             return [self.forward_layer.compute_output_shape(input_shape)] * 2
 
-    def call(self, inputs, mask=None):
-        y = self.forward_layer.call(inputs, mask)
-        y_rev = self.backward_layer.call(inputs, mask)
+    def call(self, inputs, training=None, mask=None):
+        y = self.forward_layer.call(inputs, mask=mask, training=training)
```

Comment:
Better to check whether the layer's call method accepts the `training` and `mask` arguments.